nf_core/pipeline-template/nextflow.config

```diff
@@ -233,6 +234,10 @@ dag {
     enabled = true
     file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
 }
+prov {
+    enabled = true
+    file    = "${params.outdir}/pipeline_info/manifest_${trace_timestamp}.bco.json"
+}
```

Comment:
```suggestion
prov {
    enabled = true
    format = 'bco'
    file    = "${params.outdir}/pipeline_info/manifest_${trace_timestamp}.bco.json"
}
```

Reply:
Or just wait a little while, I will make BCO the default in 1.2

Reply:
You don't have to, I will try to keep things backwards compatible

Reply:
when is 1.2 coming in?

Reply:
Probably next week

Reply:
```suggestion
prov {
    enabled = true
    formats {
        bco {
            file = "${params.outdir}/pipeline_info/manifest_${trace_timestamp}.bco.json"
        }
    }
}
```